### DISCUSSION TITLE
I can not get Student benefits

### DISCUSSION THREAD

**Discussion by 2ssunny** [community/community#112245] [GitHub Education]
### Hi everyone,

Hi. My application for Github education was submitted and approved at March 7, 2024 but I could not get global student benefits, yet.
On the website, it says that I will get benefits in 3 day, but I could not get any benefit. Actually, I renewed my student verification, and my school and document used to verify. Is this the reason my benefit is delayed?
I hope you check it and reply when I can get benefit, as soon as possible.

**Comment by rahman-cmd** [comment]
contact support

**Comment by 2ssunny** [comment]
I just got a email and my Github global campus has been confirmed.
